packages/minato/readme.md

```diff
@@ -20,7 +20,7 @@ Type Driven Database Framework.
 | Driver | Version | Notes |
 | ------ | ------ | ----- |
 | [MySQL](https://github.com/shigma/minato/tree/master/packages/mysql) | [![npm](https://img.shields.io/npm/v/@minatojs/driver-mysql?style=flat-square)](https://www.npmjs.com/package/@minatojs/driver-mysql) | MySQL 5.7, MySQL 8.0, MariaDB 10.5 |
-| [MongoDB](https://github.com/shigma/minato/tree/master/packages/mongodb) | [![npm](https://img.shields.io/npm/v/@minatojs/driver-mongo?style=flat-square)](https://www.npmjs.com/package/@minatojs/driver-mongo) | |
+| [MongoDB](https://github.com/shigma/minato/tree/master/packages/mongo) | [![npm](https://img.shields.io/npm/v/@minatojs/driver-mongo?style=flat-square)](https://www.npmjs.com/package/@minatojs/driver-mongo) | |
```

Comment:
LGTM

BTW, is there a version range for MongoDB?

Reply:
It depends on the operator that you use in your query.

Reply:
I mean, the minimum version that allows all operations of the current database.